packages/react-components/src/components/ParticipantList.tsx

```diff
@@ -258,6 +278,9 @@ export const ParticipantList = (props: ParticipantListProps): JSX.Element => {
   const displayedParticipants: ParticipantListParticipant[] = useMemo(() => {
     return onRenderParticipant ? participants : getParticipantsForDefaultRender(participants, excludeMe, myUserId);
   }, [participants, excludeMe, myUserId, onRenderParticipant]);
+
+  sortParticipants(displayedParticipants as CallParticipantListParticipant[]);
```

Comment:
this is being cast as `CallParticipantListParticipant` but there's no guarantee that's correct.
Meaning in the sort function `a.raisedHand.raisedHandOrderPosition` could throw an exception if a.raisedHand is undefined

Reply:
Agree, moved type into sorting